yeonjae/programmers-1/예산.js

```diff
@@ -1,5 +1,7 @@
 function solution(d, budget) {
-  const departmentBudget = d.sort((a, b) => {
+  const departmentBudget = [];
+
+  departmentBudget = d.sort((a, b) => {
     return a - b;
   });
```

Comment:
```suggestion
  departmentBudget = d.sort((a, b) => a - b);
```
- 한 줄로 표현하는게 깔끔해보입니다.